apisix/plugins/ip-restriction.lua

```diff
@@ -27,40 +27,29 @@ local lrucache  = core.lrucache.new({
 
 local schema = {
     type = "object",
-    oneOf = {
-        {
-            title = "whitelist",
-            properties = {
-                whitelist = {
-                    type = "array",
-                    items = {anyOf = core.schema.ip_def},
-                    minItems = 1
-                },
-                message = {
-                    type = "string",
-                    default = "Your IP address is not allowed"
-                },
-            },
-            required = {"whitelist"},
-            additionalProperties = false,
+    properties = {
+        message = {
+            type = "string",
+            minLength = 1,
+            maxLength = 64,
```

Comment:
64 looks too short to me. What about 4096?

Reply:
Would 4096 be too long?What about 1024?😃